src/chapters/data-types/exercises.rst

```diff
@@ -0,0 +1,64 @@
+Exercises: Data Types
+=====================
+
+To get started, open up Visual Studio and create a new console program by clicking *New* and selecting *Console Application* under *.Net Core App*.
+
+For each part of the exercises, carefully note in code comments which part of the exercises you are working on. 
+When you are done with that particular section of the exercises, comment out your solution and start working on the next one!
+
+#. **Input/output**: Write a new "Hello, World" program to prompt the
+   user for their name and greet them by name.
+
+   #. Add a question to ask the user:
+
+      .. sourcecode:: csharp
+
+         Console.WriteLine("What is your name?");
+
+   #. Create a variable to store the user's input:
+
+      .. sourcecode:: csharp
+
+         string name = Console.ReadLine(); 
+
+   #. Use concatenation to print the greeting:
+
+      .. sourcecode:: csharp
+
+         Console.WriteLine("Hello " + name);
+
+   #. Click on the arrow in the top left to run your program.
```

Comment:
again, id be less specific about the arrow's location until we're more clear on the similarities/differences on windows

Reply:
Done! Thanks!